src/sagas/filtersAside.js

```diff
@@ -60,16 +107,18 @@ export function* fetchStat(action) {
       countL && requests.push(call(Api.labelsStats, paramsPart));
     }
 
+    lessonAsCollection && requests.push(call(Api.collectionsStats, prepareDailyLessonParams({ ...params })));
```

Comment:
Basically it will issue another call to stats?
But if this is the case you should ignore another call to stats? Not?

Reply:
I merge results of two calls